src/commands/cronjob/create.tsx

```diff
@@ -97,18 +87,28 @@ export class Create extends ExecRenderBaseCommand<typeof Create, Result> {
       throw new Error("no project found for app installation");
     }
 
+    const destination: CronjobCreationData["destination"] =
+      buildCronjobDestination(url, command, interpreter) ??
+      (() => {
+        throw new Error(
+          "Url or command and interpreter combination could not be parsed.",
+        );
+      })();
```

Comment:
This looks a bit awkward. Maybe move the `throw` statement directly to `buildCronjobDestination`?